source/_integrations/bzutech.markdown

```diff
@@ -0,0 +1,17 @@
+---
+title: Bzu Tech
+description: Instructions on how to integrate your Bzu Cloud sensors into Home Assistant.
+ha_category:
+  - Sensor
+ha_release: 2024.1
+ha_config_flow: true
+ha_iot_class: Cloud Polling
+ha_domain: bzutech
+ha_platforms:
+  - sensor
+ha_integration_type: integration
+---
+
+The **Bzu Tech** {% term integration %} allows you to integrate your [Bzu Cloud](https://canary.is) sensors in Home Assistant.
```

Comment:
```suggestion
The **Bzu Tech** {% term integration %} allows you to integrate your [Bzu Cloud](https://canary.is) sensors in Home Assistant.
```
Do I need an account and login credentials as a prerequisite?

Reply:
> Do I need an account and login credentials as a prerequisite?

Yes

Reply:
would this be worth mentioning? as a prerequisite?

Reply:
> would this be worth mentioning? as a prerequisite?

added having an account with registered devices as prerequisites